web_custom_label/models/ir_ui_view.py

```diff
@@ -0,0 +1,47 @@
+# © 2018 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from lxml import etree
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using etree to parse untrusted XML data is known to be vulnerable to XML attacks. Replace etree with the equivalent defusedxml package.](https://app.codacy.com/manual/numigi/odoo-web-addons/pullRequest?prid=4960433)